### PR TITLE
Updated Restaurant Views

### PR DESCRIPTION
**Pull request by Jackimaru96**
- Updated All restaurant views
- Updated individual restaurant views
- Updated SQL table (OwnedRestaurants) and included more data

### CODE REVIEW

File: controllers/restaurants.controller.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const db = require('../db/index');
+
+var bcrypt = require('bcryptjs');
+var { check, validationResult } = require('express-validator');
+var passport = require('passport');
```
Comment (wailu) [inline controllers/restaurants.controller.js]: Can remove these as it's not used here.

File: views/components/navbar.pug [hunk 2]
```diff
@@ -9,7 +9,7 @@ mixin navbar
     .collapse.navbar-collapse(id='navbarSupportedContent')
       ul.navbar-nav.mr-auto
         li.nav-item
-          a.nav-link(href='#') All restaurants
+          a.nav-link(href='../restaurants') All restaurants
```
Comment (wailu) [inline views/components/navbar.pug:12]: Does this work? I think it should be `href='/restaurants'`

**Review by wailu — APPROVED** [review]
Looks fine overall; I left just a few comments!

For the intellij files, you can try to
1. add `.idea/*` into `.gitignore`
2. do `git rm -r .idea`
3. then commit the changes using `git commit -m 'Update .gitignore'`

See if this works!

File: views/components/navbar.pug (hunk 2 above)
Comment (Jackimaru96) [reply views/components/navbar.pug:12]: Yeah this works on my com

File: controllers/restaurants.controller.js (hunk 1 above)
Comment (Jackimaru96) [reply controllers/restaurants.controller.js]: Removed

**Pull request merged**